src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -218,7 +218,10 @@ def __call__(self, poller):
                 correlation_message = 'Correlation ID: {}'.format(correlation_id)
             except:  # pylint: disable=bare-except
                 pass
-            self._template_progress(correlation_id)
+            try:
+                self._template_progress(correlation_id)
+            except Exception as ex:  # pylint: disable=broad-except
+                logger.info('%s during progress reporting: %s', getattr(type(ex), '__name__', type(ex)), ex)
```

Comment:
I assume we will make change to disable the insights call unless the logging level is `verbose+`? If yes, we should log at a `warning` level, considering this is an exception regardless, though not fatal.